ontologies/src/main/resources/application.conf

```diff
@@ -1,4 +1,4 @@
 groundinginsight {
-    ontologyPath = ${ontologies.path}/CompositionalOntology_metadata.yml
+//    ontologyPath = ${ontologies.path}/CompositionalOntology_metadata.yml
```

Comment:
We have to do something about this eventually.  This setting in eidoscommon is depending on the configuration of eidos even though the project dependencies go the other way around.

Reply:
@kwalcock I just pushed a fix for this, thanks for noticing

Reply:
I *think* that ${EidosSystem.domain} is just as much from the main project and will cause the same problem.  It doesn't matter until the ontologies subproject is used on its own, like during testing or maybe publication, and this application.conf needs to be processed on its own.  Something or another complained at me and pointed to this as the cause and I had to agree with it.  It can be addressed in a different PR if you don't mind the line being commented out.

Reply:
If you are still have this branch handy, could you uncomment the line that was fixed?

Reply:
I think i did what you were asking, if I misunderstood please just let me know